### PR TITLE
MIME4J-271 Make possible to define a Content-Type parameter

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: dom/src/main/java/org/apache/james/mime4j/message/DefaultMessageBuilder.java [hunk 1]
```diff
@@ -248,6 +250,10 @@ public Multipart newMultipart(final String subType) {
         return new MultipartImpl(subType);
     }
 
+    public Multipart newMultipart(final String subType, NameValuePair... contentTypeParameters) {
```
Comment (chibenwa) [inline dom/src/main/java/org/apache/james/mime4j/message/DefaultMessageBuilder.java:253]: final parameter  spotted...

File: dom/src/main/java/org/apache/james/mime4j/message/MultipartImpl.java [hunk 2]
```diff
@@ -35,11 +39,15 @@
...
-     * Creates a new empty <code>Multipart</code> instance.
-     */
     public MultipartImpl(String subType) {
-        super(subType);
+        this(subType, Collections.<NameValuePair>emptyList());
+    }
+
+        /**
+         * Creates a new empty <code>Multipart</code> instance.
+         */
```
Comment (chibenwa) [inline dom/src/main/java/org/apache/james/mime4j/message/MultipartImpl.java]: Indent fail?

File: dom/src/main/java/org/apache/james/mime4j/message/DefaultMessageBuilder.java (hunk 1 above)
Comment (chibenwa) [reply dom/src/main/java/org/apache/james/mime4j/message/DefaultMessageBuilder.java:253]: I'm too zealous, I guess.... Let's stick to MIME4J coding style...

**Review by chibenwa — APPROVED** [review]

**Review by mbaechler — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**